modules/AOW_WorkFlow/aow_utils.php

```diff
@@ -564,7 +579,10 @@ function getModuleField($module, $fieldname, $aow_field, $view='EditView',$value
     $ss->assign("MOD", $mod_strings);
     $ss->assign("APP", $app_strings);
 
-    //$return = str_replace($fieldname,$ss->fetch($file));
+    //In the case of a csv being generated check if the field type is a checkbox. if so just return the value
+    if($report_csv && $vardefFields[$fieldname]['type'] === 'bool') {
```

Comment:
@shogunpol , in function parameter you put value name `$reports_csv` but in if statement you check for `$report_csv` (missing 's' report**s**_csv)